packages/syft/src/syft/service/code/user_code.py

```diff
@@ -348,6 +349,18 @@ def _coll_repr_(self) -> dict[str, Any]:
             "Submit time": str(self.submit_time),
         }
 
+    @property
+    def user(self) -> UserView | SyftError:
```

Comment:
is this correct? doesnt this just get the user for the current client? What if the current client is the admin, but the DS created this user code?